pkgs/test/src/cli.ts

```diff
@@ -11,9 +13,46 @@ cli.command("launch")
     .description("Launch the Redo app")
     .action(async () => {
         console.log("Launching the app...")
+        const { outdated } = await isNewVersionAvailable(version)
+        if (outdated) { console.log("New Update is Avaiable. You can Upgrade to the latest version with redo update.") }
         shell(await getPath(version))
     })
+cli.command("upgrade")
+    .description("Upgrade Redo to the latest available version")
+    .action(async () => {
+        const { outdated, release } = await isNewVersionAvailable(version)
+        if (outdated) {
+            shell(await getPath(release!))
+            console.log("Redo has been updated to " + release)
+        } else {
+            console.log("Redo is up to date!")
+        }
+    })
+//TODO: move logic elsewhere to clean file up
+cli.command("clean")
+    .description("Remove all instances of redo older than version found in package.json")
+    .action(async () => {
+        console.log(`Removing any versions of redo older than ${version}`)
+        const REDO_DIR = ensureRedoDir()
+        readdir(REDO_DIR, (_, content) => {
```

Comment:
Let's prompt the user either once for all versions that will be deleted or for each version that will be deleted to avoid breaking stuff